### DISCUSSION TITLE
Use diskstation to use as VDI for Proxmox

### DISCUSSION THREAD

**Discussion by RudiOnTheAir** [Thinstation/thinstation#871] [Ideas]
Maybe there is already a way to do this. But it would be nice to use a spice virtviewer with integrated VDI client from here..

https://github.com/joshpatten/PVE-VDIClient

Tnx for some hints...

**Comment by Doncuppjr** [comment]
Most of what you would need is here, https://github.com/Thinstation/thinstation/wiki/Packaging
Looks like a pretty simple package.

**Comment by Doncuppjr** [comment]
Kind of weird he didn't just provide a freeze file.

**Comment by Doncuppjr** [comment]
Also, a SVG would be nice instead of a single PNG

**Reply by RudiOnTheAir** [reply]
U mean the maintainer of the VDI Client?

**Reply by Doncuppjr** [reply]
Yes

**Reply by RudiOnTheAir** [reply]
Hello
Yes, I can't say anything about that. Unfortunately, the company behind Proxmox doesn't currently have its own client that is based on Spice. There is also the question of how long Spice will remain a part of the product. I heard that it is no longer being developed. Well, we'll have to wait and see.
